benchmarks/markdown/benches/gen_benches.nit

```diff
@@ -0,0 +1,294 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Generate bench suites for markdown parsers.
+module gen_benches
+
+import opts
+
+# Decorate a markdown plain text for bench puposes.
+#
+# *This is not the markdown you are looking for.*
+interface MarkdownDecorator
+
+	# Parse the text and decorate it.
+	# Behavior depends on `LineDecorator` and `WordDecorator`.
+	fun decorate_text(txt: Text): Text is abstract
+
+	# Is `c` a something else than a letter a digit or a `-`?
+	fun is_word_break(c: Char): Bool do
+		if not c.is_letter or c.is_digit or c == '-' then
```

Comment:
Doesn't `not is_letter` cover the other 2 cases? and this condition could be returned directly.